cmd/zetae2e/local/bitcoin.go

```diff
@@ -14,6 +14,11 @@ import (
 	"github.com/zeta-chain/node/testutil"
 )
 
+const (
+	groupDeposit  = "btc_deposit"
+	groupWithdraw = "btc_withdraw"
```

Comment:
What group mean here?

Reply:
Renamed as `testGroupDepositName` and `testGroupWithdrawName` testGroupWithdrawName https://github.com/zeta-chain/node/pull/3417/commits/142e0a459eb2f993a014328f95c5cc43a7f385ab